android-app/app/src/main/java/de/dhbw/ka/se/fibo/CreateAccountActivity.java

```diff
@@ -55,9 +97,95 @@ private void initializeButton() {
     }
 
     private void registerButtonClicked() {
-        Intent i = new Intent(CreateAccountActivity.this,
+        i = new Intent(CreateAccountActivity.this,
                 MainActivity.class);
-        startActivity(i);
-        finish();
+        create_account_email = findViewById(R.id.create_account_email);
+        String email = create_account_email.getText().toString();
+        passwordField = findViewById(R.id.create_account_password_layer);
+        emailField = findViewById(R.id.create_account_email_layer);
+        create_account_password = findViewById(R.id.create_account_password);
+        String password = create_account_password.getText().toString();
+        createUser(email, password);
+
+    }
+
+    private void createUser(String email, String password) {
+        String url = "http://10.0.2.2:8000/users/register/";
+
+        RequestQueue queue = Volley.newRequestQueue(this);
+
+        System.out.println("Email: " + email);
+        System.out.println("Passwort: " + password);
+
+
+        if (!checkValidInput(email, password)) {
+            return;
+        }
+        StringRequest request = new StringRequest(Request.Method.POST, url, response -> {
+
+            Toast successToast = Toast.makeText(this, "Success", Toast.LENGTH_LONG);
+            successToast.setGravity(Gravity.TOP, 0, 0);
+            successToast.show();
+            System.out.println("Successfully" + response);
+            startActivity(i);
+            finish();
+
+        }, error -> {
+
+            if (error.networkResponse != null) {
+                switch (error.networkResponse.statusCode) {
+                    case 400:
+                        System.out.println("Bad Request");
+                        break;
+                    case 401:
+                        System.out.println("Unauthorized");
+                        break;
+                    case 404:
+                        System.out.println("Not Found");
+                        break;
+                    case 500:
+                        System.out.println("Internal Server Error");
+                        break;
+                    default:
+                        break;
+                }
```

Comment:
It'd be great to let the user know that something went wrong. The user does not notice the STDOUT logs. (Also, if they should be kept, please use `Log.e` as the appropiate logger instead.)